examples/analog_out/gen_voltage_sample.py

```diff
@@ -0,0 +1,13 @@
+"""Example of analog output voltage generation.
+
+This example demonstrates how to output a single Voltage Update
+(Sample) to an Analog Output Channel.
+"""
+
+import nidaqmx
+
+with nidaqmx.Task() as task:
+    task.ao_channels.add_ao_voltage_chan("Dev1/ao0")
+
+    print(f"Generate {task.write(1.1)} voltage sample.")
+    task.stop()
```

Comment:
unnecessary since you didn't explicitly call start

Reply:
Noted, `stop` is removed.